### DISCUSSION TITLE
TypeError: n.auth is not a function

### DISCUSSION THREAD

**Discussion by marcellokabora** [angular/angularfire#3227] [Q&A]
Hello Devs, Im having this error after I deploy my app but not in development mode, can anybody help me?

Error: `TypeError: n.auth is not a function`

I am using this providers: `{ provide: AUTH_SETTINGS, useValue: { appVerificationDisabledForTesting: true } }`

**Comment by Rahemanali-24** [comment]
have you find this error solution?

**Reply by pahtz** [reply]
I have the TypeError with 17.0.1 also. I had to downgrade to Angular 16.x.x and @angular/fire@7.6.1
